packages/wasm-dpp/test/unit/document/Document.spec.js

```diff
@@ -207,238 +239,200 @@ describe('Document', () => {
 
       document = new Document(rawDocument, dataContract);
 
-      expect(document.getUpdatedAt().getTime()).to.equal(rawDocument.$updatedAt);
+      expect(document.getUpdatedAt()).to.equal(rawDocument.$updatedAt);
     });
-  });
-
-  describe('#getId', () => {
-    it('should return ID', () => {
-      const id = '123';
 
-      document.id = id;
+    describe('#getId', () => {
```

Comment:
I'm not sure why the indent did change here

Reply:
good eye, fixed